gnucash/report/reports/standard/taxinvoice.scm

```diff
@@ -87,9 +87,9 @@
 (define optname-jobnumber-show		(N_ "Show Job number"))
 (define optname-netprice		(N_ "Show net price"))
 (define optname-invnum-next-to-title	(N_ "Invoice number next to title"))
-(define optname-border-collapse		(N_ "table-border-collapse"))
-(define optname-border-color-th		(N_ "table-header-border-color"))
-(define optname-border-color-td		(N_ "table-cell-border-color"))
+(define optname-border-collapse		"table-border-collapse")
+(define optname-border-color-th		"table-header-border-color")
+(define optname-border-color-td		"table-cell-border-color")
```

Comment:
Nope, these are option names in the Display tab. They need to be translated. They'd look nicer capitalized, as would the row and column ones above.